File: app/src/main/java/com/happs/ximand/ringcontrol/model/object/command/simple/MakeRingCommand.java
package com.happs.ximand.ringcontrol.model.object.command.simple;

public class MakeRingCommand extends SimpleBluetoothCommand {

    private static final byte MAKE_RING_COMMAND = 31;

    public MakeRingCommand(byte seconds) {
        super(MAKE_RING_COMMAND, seconds);
    }

}

File: app/src/main/java/com/happs/ximand/ringcontrol/model/object/command/simple/MakeRingCommand.kt
package com.happs.ximand.ringcontrol.model.`object`.command.simple

class MakeRingCommand(seconds: Byte) : SimpleBluetoothCommand(MAKE_RING_COMMAND, seconds) {
    companion object {
        private const val MAKE_RING_COMMAND: Byte = 31
    }

}